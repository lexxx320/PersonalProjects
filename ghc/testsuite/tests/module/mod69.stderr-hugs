ERROR "mod69.hs":3 - Illegal `@' in expression
