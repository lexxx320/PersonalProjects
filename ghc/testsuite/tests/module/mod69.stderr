
mod69.hs:3:7: Pattern syntax in expression context: x@1
